src/App.jsx

```diff
@@ -1,9 +1,99 @@
 import './App.css';
+import soldProducts from "./helpers/soldProducts.js";
+import productStock from "./helpers/productStock.js";
+import productsToSell from "./helpers/productsToSell.js";
+import tvNameToString from "./helpers/tvNameToString.js";
+import {bestSellingTv, inventory} from "./constants/inventory.js";
+import priceFormatter from "./helpers/priceFormatter.js";
+import screensizeFormatter from "./helpers/screensizeFormatter.js";
+import checkIcon from "./assets/check.png";
+import minusIcon from "./assets/minus.png";
+
 
 function App() {
-  return (
-    <h1>Begin hier met met maken van de applicatie!</h1>
-  )
+
+    function handleClickMostSoldFirst() {
+        console.log("Meest verkocht eerst")
+    }
+
+    function handleClickCheapestFirst() {
+        console.log("Goedkoopste eerst")
+    }
+
+    function handleClickBestForSportsFirst() {
+        console.log("Meest geschikt voor sport eerst")
+    }
+
+    return (
+        <main>
+            <header>
+                <h1>Tech it easy dashboard</h1>
+            </header>
+
+
+            <h2> Verkoopoverzicht </h2>
+
+            {/*Wrapper voor de 3 boxen */}
+            <div className="salesWrapper">
```

Comment:
Een div met meerdere sections is niet heel netjes. Een section met meerdere articles daarentegen is dat wel.